Fix aclocal warnings:
/usr/share/aclocal/ORBit.m4:4: warning: underquoted definition of AM_PATH_ORBIT
/usr/share/aclocal/libIDL.m4:6: warning: underquoted definition of AM_PATH_LIBIDL
--- ORBit-0.5.17/ORBit.m4
+++ ORBit-0.5.17/ORBit.m4
@@ -3,3 +3,3 @@
 dnl
-AC_DEFUN(AM_PATH_ORBIT,
+AC_DEFUN([AM_PATH_ORBIT],
 [dnl 
--- ORBit-0.5.17/libIDL/libIDL.m4.in
+++ ORBit-0.5.17/libIDL/libIDL.m4.in
@@ -5,3 +5,3 @@
 dnl
-AC_DEFUN(AM_PATH_LIBIDL,
+AC_DEFUN([AM_PATH_LIBIDL],
 [dnl 
